### ISSUE TITLE
Is there any upstreaming plans?

### ISSUE THREAD

**Issue by mrkiko** [ulli-kroll/rtw88-usb#23]
If not, what are the roadblocks?

**Comment by ulli-kroll** [comment]
roadblockers none.
But this driver in (currently) not reliable because of this
#8 
found this by accident one week ago, in some other source

**Comment by antermin** [comment]
See also: [patch series](https://lore.kernel.org/lkml/20220518082318.3898514-1-s.hauer@pengutronix.de/) submitted to upstream by @saschahauer

**Comment by ruantu** [comment]
upstream merged, in linux 6.2

**Comment by mrkiko** [comment]
I was trying to search for my hardware ID (the USB dongle I added in 2fe3561af5a780e3c5c3f9892148b0ffb6173a1c ) but can't find it.
Am I grepping wrong?

**Comment by mrkiko** [comment]
Sorry, was unclear. I was grepping for the USB idVendor / idProduct in the linux kernel tree, but can't find them.
Was curious to know if upstream support is being merged for that chipset as well.

**Comment by saschahauer** [comment]
Yes, that was merged as well. See https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/drivers/net/wireless/realtek/rtw88/rtw8822bu.c#n52

**Comment by mrkiko** [comment]
On Mon, 9 Jan 2023, Sascha Hauer wrote:

> Date: Mon, 9 Jan 2023 09:31:12
> From: Sascha Hauer ***@***.***>
> Reply-To: ulli-kroll/rtw88-usb
>     ***@***.***>
> To: ulli-kroll/rtw88-usb ***@***.***>
> Cc: mrkiko ***@***.***>, Author ***@***.***>
> Subject: Re: [ulli-kroll/rtw88-usb] Is there any upstreaming plans? (Issue
>     #23)
> 
> 
> Yes, that was merged as well. See https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/drivers/net/wireless/realtek/rtw88/rtw8822bu.c#n52
> 
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.[AEA6HOJJV4SR77IAI3DT72TWRPEFBA5CNFSM5GGP4D72YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOKH4NWCQ.gif] Message ID:
> ***@***.***>
> 
> 
> 
Thanks a lot to all involved.
Seeing that hw supported is really great.

**Comment by peterhoeg** [comment]
Just for my understanding - with most upstreamed, is the purpose of this repo to act as a staging environment for new features/fixes/devices or is everything expected to happen upstream?
